tests/src/Glpi/Tests/CommonTestCase.php

```diff
@@ -145,10 +144,7 @@ protected function loginWithUserToken($userToken) {
     * Get a unique random string
     */
    protected function getUniqueString() {
-      if (is_null($this->str)) {
-         return $this->str = uniqid('str');
-      }
-      return $this->str .= 'x';
+      return uniqid('str', true);
```

Comment:
you should not change this implementation. It is picked from GLPI's unit tests and having the same implementation in both projects gives a change to mutualize the testing tools.